scipy/stats/qmc.py

```diff
@@ -233,3 +233,4 @@
 
 """
 from ._qmc import *  # noqa: F403
+from ._qmc import __all__  # noqa: F403
```

Comment:
Instead of `*`? Isn't it redundant then? But sure no objections otherwise 😃 Whatever works as long we don't change the imports and I can still do `sp.stats.qmc.Sobol`

Reply:
what is the error without this change? Pretty sure this isn't the ideal solution.